Lib/pdb.py

```diff
@@ -562,6 +562,12 @@ def do_commands(self, arg):
             except:
                 self.error("Usage: commands [bnum]\n        ...\n        end")
                 return
+        try:
+            self.get_bpbynumber(bnum)
+        except ValueError as err:
+            self.error(err)
```

Comment:
```suggestion
            self.error('cannot set commands: %s' % err)
```

Reply:
As it already displays a more informative error, why do you think a more generic one would work better?

Reply:
This includes the original error as well, so it's not more generic.  The prefix makes it clear that what you asked for did not happen.

Reply:
you are right, good point :)